Cache/Redis.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace Smartive\HandlebarsBundle\Cache;
+
+use Handlebars\Cache;
+use Predis\Client;
+use Psr\Log\LoggerInterface;
+use Psr\Log\NullLogger;
+
+/**
+ * Client to cache Handlebars templates using Redis.
+ */
+class Redis implements Cache
```

Comment:
i guess for now its ok to commit this class in here so we can move forward, but we should also give it upstream to xamin. we don't use any symfony specific things in here as far as i can see.